src/main/java/org/folio/validate/definition/SingleSelectDropdownValidator.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.validate.definition;
+
+import static org.folio.rest.jaxrs.model.CustomField.Type.SINGLE_SELECT_DROPDOWN;
+
+import org.springframework.stereotype.Component;
+
+import org.folio.rest.jaxrs.model.CustomField;
+
+@Component
+public class SingleSelectDropdownValidator extends SelectableField implements Validatable {
```

Comment:
As I understand, SINGLE_SELECT_DROPDOWN and RADIO_BUTTON types only differ in how they are displayed on UI, so one validator can be used for both of them

Reply:
The difference is allowed options, for SINGLE_SELECT_DROPDOWN - 200, RADIO_BUTTON - 5, agree that validator can be used for both types

Reply:
Ah, ok then